public/app/features/panel/panellinks/linkSuppliers.ts

```diff
@@ -1,8 +1,31 @@
 import { PanelModel } from 'app/features/dashboard/state/PanelModel';
-import { FieldDisplay, ScopedVars, DataLinkBuiltInVars } from '@grafana/ui';
-import { LinkModelSupplier, getTimeField } from '@grafana/data';
+import { FieldDisplay, ScopedVars, ScopedVar } from '@grafana/ui';
+import { LinkModelSupplier, getTimeField, Labels } from '@grafana/data';
 import { getLinkSrv } from './link_srv';
 
+interface SeriesVars {
+  name: string;
```

Comment:
DataFrame name is optional... or will this fill things in with a default?
same with labels...